src/test/scala/com/fulcrumgenomics/umi/GroupReadsByUmiTest.scala

```diff
@@ -267,6 +269,10 @@ class GroupReadsByUmiTest extends UnitSpec with OptionValues with PrivateMethodT
 
       hist.toFile.exists() shouldBe true
 
+      // TODO: Create a more comprehensive test case
```

Comment:
```suggestion
      // TODO: Consider creating more unit tests that vary the following metric fields
```

Reply:
@znorgaard how about adding to existing tests?  E.g. https://github.com/fulcrumgenomics/fgbio/blob/e1dcbabb6ad8dadc825537ef22914b1872e85337/src/test/scala/com/fulcrumgenomics/umi/GroupReadsByUmiTest.scala#L426

Reply:
@nh13, just add the metric generation and check to those other tests? 

That would work. I generally like to see a "complex" test case where non-zero values are generated for each metric entry at once. But, in this specific case we're pulling from existing variables, so individual tests might be good enough.